migration/rack_migrate/rack_migrate

```diff
@@ -160,6 +161,13 @@ if len(commits_to_consider_in_chronological_order) == 0:
     sys.exit(1)
 
 
+def markdown_link_to_commit(
+    commit_id: str,
+    link_text: str,
+) -> str:
+    return f"[{link_text}](https://github.com/ge-high-assurance/RACK/commit/{commit_id})"
```

Comment:
Is it possible for `link_text` to contain a `]`?

Reply:
I think it might, good catch!